vital/types/detected_object.h

```diff
@@ -121,6 +143,28 @@ class VITAL_EXPORT detected_object
    */
   void set_bounding_box( const bounding_box_d& bbox );
 
+
+  /**
+   * @brief Get geo_point from this detection.
+   *
+   * The geo_point for this detection is returned. This point is in
+   * world coordinates. A null geo_point
+   * is returned if no location has been supplied for this detection.
+   *
+   * @return The bounding box.
+   */
+  geo_point_sptr& location();
```

Comment:
(**API**) Are you *sure* you want to return this by reference? (And if so, why not the `const` overload also?)